test/unit/mixins/state.spec.js

```diff
@@ -1,9 +1,19 @@
-describe('State Class', function() {
+import Marionette from 'backbone.marionette';
+import StateMixin from '../../../src/mixins/state';
+
+describe('StateMixin', function() {
   beforeEach(function() {
     this.MyModel = Backbone.Model.extend({
       foo: 'bar'
     });
-    this.MyStateClass = Marionette.Toolkit.StateClass.extend({
+
+    this.StateClass = Marionette.Object;
```

Comment:
you should `.extend()` Object here.. Otherwise you're mutating Object's prototype on the next line.